### PR TITLE
[ElasticDL] Verify SELECT fields from DB and add warning on COLUMN clause

### PR DESCRIPTION
**Pull request by terrytangyuan**
Part of #664.

Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

### CODE REVIEW

File: sql/codegen_elasticdl.go [hunk 1]
```diff
@@ -149,6 +149,16 @@ func newElasticDLTrainFiller(pr *extendedSelect, db *DB, session *pb.Session, ds
...
 		return nil, err
 	}
+	hasFeatureColumns := false
+	for _, columns := range resolved.FeatureColumns {
+		if len(columns) > 0 {
+			hasFeatureColumns = true
+		}
+	}
+	if hasFeatureColumns {
+		log.Warnln("COLUMN clause is ignored since ElasticDL does not support feature columns yet")
```
Comment (tonyyang-svail) [inline sql/codegen_elasticdl.go:162]: Without the support of the feature column, how does EDL infer the shape of the input?

Comment (terrytangyuan) [reply sql/codegen_elasticdl.go:162]: Marking this as WIP for now. I'll reuse `verifier` to get this kind of information instead.

File: sql/codegen_elasticdl.go [hunk 2]
```diff
@@ -63,18 +64,20 @@ type elasticDLModelSpec struct {
...
-			features = append(features, feature)
+
+	featureNames := make([]string, 0, len(fts))
+	for featureName := range fts {
+		if featureName != pr.label {
+			featureNames = append(featureNames, featureName)
 		}
 	}
-	return features, nil
+	sort.Strings(featureNames)
```
Comment (tonyyang-svail) [inline sql/codegen_elasticdl.go:79]: I am curious about the necessity of sorting.

**Review by tonyyang-svail — APPROVED** [review]

File: sql/codegen_elasticdl.go (hunk 2 above)
Comment (terrytangyuan) [reply sql/codegen_elasticdl.go:79]: It simply makes the testing easier. Otherwise, the generated feature information string varies.

**Pull request merged**